mlir/include/mlir/Dialect/Arith/IR/ArithOps.td

```diff
@@ -1195,6 +1195,14 @@ def Arith_ExtFOp : Arith_FToFCastOp<"extf"> {
   }];
   let hasVerifier = 1;
   let hasFolder = 1;
+  let hasCanonicalizer = 1;
+
+  let arguments = (ins FloatLike:$in, DefaultValuedAttr<
+                       Arith_FastMathAttr,
+                       "::mlir::arith::FastMathFlags::contract">:$fastmath);
+  let results = (outs FloatLike:$out);
+
+  let assemblyFormat = "$in attr-dict `:` type($in) `to` type($out)";
```

Comment:
```
(`fastmath` `` $fastmath^)?
```
like in the binary operations.

Also, `DeclareOpInterfaceMethods<ArithFastMathInterface>`